docs/README.md

```diff
@@ -255,3 +256,33 @@ contribution](https://trino.io/development/process.html).
    Docker](https://www.youtube.com/watch?v=y58sb9bW2mA) gives you a starting
    point for setting up a test system on your laptop.
 
+## Docker container
+
+The build of the docs uses a Docker container that includes Sphinx and the
+required libraries. The container is referenced in the `SPHINX_IMAGE` variable
+in the `build` script.
+
+The specific details for the container are available in `Dockerfile`, and
+`requirements.in`. The file `requirements.txt` must be updated after any changes
+to `requirements.in`.
+
+The container must be published to the GitHub container registry at ghcr.io with
+the necessary access credentials and the following command:
+
+```
+docker buildx build docs --platform=linux/arm64,linux/amd64 --tag ghcr.io/trinodb/build/sphinx:7 --provenance=false --push
```

Comment:
How about replace “7” with VERSION so you can’t accidentally overwrite a real version

Reply:
Good idea

Reply:
Done